README.md

```diff
@@ -47,6 +47,9 @@ The Flatpak RuneScape client can be launched from the Jagex Launcher after insta
 > If you install the official RuneScape client it will overwrite the symbolic link needed to run the Flatpak RuneScape client<br>
 > You can create the symbolic link with the following command:<br>
 ```cd $HOME/Games/jagex-launcher/drive_c/Program\ Files\ \(x86\)/Jagex\ Launcher/Games/RuneScape/ && ln -s runescape.sh RuneScape.exe```<br>--->
+> If not installing and linking to a native installation instead, you will have to import a registry key via the wine utility `regedit`.
```

Comment:
To use a native Linux installation, they can modify either runescape.sh or runelite.sh to launch whatever application version they want. They should not need to touch the registry because the lutris script adds those for them.

 If they are using Flatpak lutris, they may need to pass environment variables via flatpak-spawn to avoid breaking single sign-on. See https://github.com/TormStorm/jagex-launcher-linux/blob/main/resources/runescape.sh#L7-L16 for an example. 

If they want to install the client inside of wine, then they will need to delete one or both of these registry entries https://github.com/TormStorm/jagex-launcher-linux/blob/main/resources/jagexlauncher.yml#L59-L68 and then they can click the install button inside the launcher to run inside of Wine.